tailwind.config.js

```diff
@@ -11,6 +11,14 @@ module.exports = {
         "gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))"
+      },
+      colors: {
+        "starick-green": "var(--starick-green)",
```

Comment:
I think it might be better to define the hex codes for the colours directly here rather than in globals.css (and remove them from globals.css)

Reply:
Done!